community-membership.md

```diff
@@ -296,5 +297,69 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and which highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
+- Has been the reviewer or author (through the sponsorship of another
+  specification sponsor) of at least 10 substantial PRs to specification or
+  related repositories
+- Demonstrated sound technical judgement, and a deep understanding of the
+  specification and the OpenTelemetry project
+- Nominated by technical committee member (
+  see [becoming a specification sponsor](#becoming-a-specification-sponsor))
+
+### Responsibilities and privileges
```

Comment:
I find it difficult to find the privileges in this list. Here are the ones I gathered:

- approve issues / PRs
- sponsor issues (see comment below)

Is that motivation enough to become a spec sponsor?

Reply:
If we continue with the spec issue management workflow we've been experimenting with, every non-trivial issue requires sponsorship. The sponsor either does the work and makes the PR themselves, or works with an outside collaborator and sponsors them. Therefore, the ability to sponsor issues means the ability to move something through the spec. 

> Is that motivation enough to become a spec sponsor?

I lifted this presentation of "Responsibilities and privileges" from the other roles which are defined in this document. This argument can be applied to any other role. Why does anyone take on these otel roles which primarily come with more responsibility? Everyone has their own reasons but I suspect an overarching one is that named roles can help you get stuff done.